src/go/supervisor/pkg/svc/install.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright © 2021 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package svc
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"golang.org/x/sys/windows/svc/eventlog"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+const supervisorInstallPath = `AppData\Local\Programs\Rancher Desktop\supervisor.exe`
+
+func InstallService(name, desc string) error {
+	instPath, err := installPath()
+	if err != nil {
+		return fmt.Errorf("getting installation path for service [%s] failed: %w", name, err)
+	}
+	m, err := mgr.Connect()
+	if err != nil {
+		return err
+	}
+	defer m.Disconnect()
+	s, err := m.OpenService(name)
+	if err == nil {
+		s.Close()
+		return fmt.Errorf("service [%s] already exists", name)
+	}
+	s, err = m.CreateService(name, instPath, mgr.Config{DisplayName: desc}, "is", "auto-started")
+	if err != nil {
+		return err
+	}
+	defer s.Close()
```

Comment:
Shouldn't we [do something](https://docs.microsoft.com/en-us/windows/win32/services/modifying-the-dacl-for-a-service) to allow the (unprivileged) user to start/stop the service?

Reply:
When would we need that use case?

Reply:
The Rancher Desktop executable is unprivileged; so in order to start the service, we need to allow it access.